web_chat.lua

```diff
@@ -260,13 +260,17 @@ end
 
 
 
---- Replaces http and https links with HTML links
+--- Removes html tags
+--- Creates a tag when http(s) links are send.
 --- It does this by selecting all the characters between "http(s)://" and a space, and puts an anker tag around it.
-local function CheckForLinks(a_Message)
+local function ParseMessage(a_Message)
 	local function PlaceString(a_Url)
 		return '<a href="' .. a_Url .. '" target="_blank">' .. a_Url .. '</a>'
 	end
-	return a_Message:gsub('http://[^%s]+', PlaceString):gsub('https://[^%s]+', PlaceString)
+	
+	a_Message = a_Message:gsub("<", "&lt;"):gsub(">", "&gt;")
+	a_Message = a_Message:gsub('http://[^%s]+', PlaceString):gsub('https://[^%s]+', PlaceString)
```

Comment:
While trying this I found that if you say `=` in the webadmin the webserver registers it as an empty string, and if you have something else in the message like `test=help` then the server (or plugin) won't receive anything at all.